pkg/exchange/okex/convert.go

```diff
@@ -277,3 +280,34 @@ func toLocalInterval(src string) string {
 		return strings.ToUpper(w)
 	})
 }
+
+func hashStringID(s string) uint64 {
+	h := fnv.New64a()
+	h.Write([]byte(s))
+	return h.Sum64()
+}
+
+func toGlobalOrder(okexOrder *okexapi.OrderDetails, isMargin bool) (*types.Order, error) {
+	timeInForce := types.TimeInForceFOK
+	if okexOrder.OrderType == okexapi.OrderTypeIOC {
+		timeInForce = types.TimeInForceIOC
+	}
+	return &types.Order{
+		SubmitOrder: types.SubmitOrder{
+			ClientOrderID: okexOrder.ClientOrderID,
+			Symbol:        okexOrder.InstrumentID,
+			Side:          types.SideType(okexOrder.Side),
```

Comment:
okexOrder.Side vlaues are BUY, SELL in upper case?

Reply:
lower case, sorry.  I used other exchange as reference, right now I use toGlobalOrders() as reference.